01-fundamentos/bloco-08-higher-order-functions-do-javascript-es6/dia-03-higher-order-functions-map-e-filter/exercicios/exercicio7.js

```diff
@@ -70,8 +70,8 @@ function authorWith3DotsOnName() {
   // escreva seu código aqui
   const threeInitialsAuthor = books.find(book => {
     const splitedName = book.author.name.split('.');
-    const howManyDots = splitedName.length;
-    return howManyDots === 4;
+    const howManyDots = splitedName.length - 1;
```

Comment:
Agora consegui ver um sentido interessante, como o ultimo nome, ou seja, o ultimo elemento de splitedName não tem um ponto final, todos os outros teriam por causa do split('.') então  o - 1 faz alcançar o valor exato de pontos que existem no nome! Aí quando for 3, como em J. R. R. Tolkien, ele retorna verdade.
Agora ta só  o ouro!

Reply:
Exato! Valeu demais pelo feedback Caio, ajudou muito!